dev.js

```diff
@@ -34,7 +34,7 @@ var loadClientBundleLink = path.join(dirs.meteor, 'client/loadClientBundle.html'
 
 var requireServerBundleJs = path.join(dirs.meteor, 'server/require.server.bundle.js');
 
-require('./core-js-custom-build');
+exec('node core-js-custom-build.js');
 
```

Comment:
because before the code has inside `exec` ( [L16](https://github.com/jedwards1211/meteor-webpack-react/pull/106/files#diff-18c0b4786d7a5998d19202140632ebb8L16) and [L18](https://github.com/jedwards1211/meteor-webpack-react/pull/106/files#diff-18c0b4786d7a5998d19202140632ebb8L18) ) in core-js-custom-build.js file.
but, now core-js-custom-build file doesn't have `exec`s then it is non-blocking.
so, I replaced `exec` instead of `require`.